src/basic/missing_sched.h

```diff
@@ -7,6 +7,10 @@
 #define CLONE_NEWCGROUP 0x02000000
 #endif
 
+#ifndef CLONE_NEWTIME
```

Comment:
for new additions to missing_*.h we would like if a new comment indicating which kernel commit and version added the flag, look for example how `missing_capability.h` does this. We are hoping that eventually we can remove these fallback definitions again, once we bump the baseline enough.

Reply:
/* 769071ac9f20b6a447410c7eaa55d1a5233ef40c (5.8) */